packages/inter-protocol/src/vaultFactory/storeUtils.js

```diff
@@ -128,15 +128,22 @@ harden(fromVaultKey);
  *
  * @param {PriceQuote} quote
  * @param {Ratio} compoundedInterest
+ * @param {Ratio} margin
  * @returns {number}
  */
-export const normalizedCollRatio = (quote, compoundedInterest) => {
+export const normalizedCollRatio = (quote, compoundedInterest, margin) => {
   const amountIn = getAmountIn(quote).value;
   const amountOut = getAmountOut(quote).value;
   const interestNumerator = compoundedInterest.numerator.value;
   const interestBase = compoundedInterest.denominator.value;
-  const numerator = multiply(amountIn, interestNumerator);
-  const denominator = multiply(amountOut, interestBase);
+  const numerator = multiply(
```

Comment:
Can we use amount math instead? It's purpose is to keep the amount units all correct rather than open-coding with raw numbers. 

Otherwise, it seems that we need (in the future) some additional helpers for e.g., doing unitless operations on Amounts.